.github/workflows/lint.yaml

```
@@ -0,0 +1,155 @@
+name: lint
+on: [push, pull_request]
+jobs:
+  check-licences:
+    name: Check Licences
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Enable Problem Matcher for GitHub annotations
+        run: echo "::add-matcher::.github/problem-matcher-lint-check-licences.json"
+      - name: Check licenses
+        shell: bash
+        run: python3 ./scripts/enforce_licence.py
```

I think installing python-is-python3 should simplify this back to:
```suggestion
        run: ./scripts/enforce_licence.py
```

---

Fixed!